src/main/java/com/quantumsoft/qupathcloud/dao/spec/StudiesPathBuilder.java

```diff
@@ -15,29 +15,46 @@
 
 package com.quantumsoft.qupathcloud.dao.spec;
 
+import static com.quantumsoft.qupathcloud.dao.Constants.DATASETS;
+import static com.quantumsoft.qupathcloud.dao.Constants.DICOM_STORES;
+import static com.quantumsoft.qupathcloud.dao.Constants.DICOM_WEB;
+import static com.quantumsoft.qupathcloud.dao.Constants.LOCATIONS;
+import static com.quantumsoft.qupathcloud.dao.Constants.PROJECTS;
+import static com.quantumsoft.qupathcloud.dao.Constants.STAGE;
+import static com.quantumsoft.qupathcloud.dao.Constants.STUDIES;
+
 import com.quantumsoft.qupathcloud.exception.QuPathCloudException;
 
-import static com.quantumsoft.qupathcloud.dao.Constants.*;
+/**
+ * StudiesPathBuilder to build a path for access to Studies in chosen Project, Location, Dataset and
+ * DICOM Store.
+ */
```

Comment:
StudiesPathBuilder builds a path to access Studies in a specified Project, Location, Dataset and DICOM Store.